docs/intelligentapps/overview.md

```diff
@@ -1,34 +1,110 @@
+| [Model Evaluation](/docs/intelligentapps/evaluation) | Comprehensive model assessment using datasets and standard metrics. Measure performance with built-in evaluators (F1 score, relevance, similarity, coherence) or create custom evaluation criteria. | ![evaluate](./images/overview/Eval.png) |
+| [Fine-tuning](/docs/intelligentapps/finetune) | Customize and adapt models for specific domains and requirements. Train models locally with GPU support or leverage Azure Container Apps for cloud-based fine-tuning. | ![fine-tune](./images/overview/Fine-tune.png) |
+| [Model Conversion](/docs/intelligentapps/modelconversion) | Convert, quantize, and optimize machine learning models for local deployment. Transform models from Hugging Face and other sources to run efficiently on Windows with CPU, GPU, or NPU acceleration. | ![conversion](./images/overview/Conversion.png) |
 
 ## Who is AI Toolkit for?
 
-Any developer who wants to explore, test, evaluate, and fine-tune generative AI models when building AI apps.
+AI Toolkit is designed for anyone working with generative AI, from beginners to experts:
+
+### Developers
+* **App developers** building AI-powered applications who need to integrate language models
+* **Full-stack developers** looking to add intelligent features to web and desktop applications
+* **Mobile developers** prototyping AI functionality before production deployment
+
+### AI Engineers & Data Scientists
+* **AI engineers** fine-tuning models for specific domains and deploying to production
+* **Data scientists** evaluating model performance and comparing different approaches
+* **ML engineers** converting and optimizing models for efficient local deployment
+
+### Researchers & Educators
+* **AI researchers** experimenting with different models and prompt engineering techniques
+* **Educators** teaching AI concepts and demonstrating model capabilities
+* **Students** learning about generative AI and hands-on model interaction
+
+### Key Use Cases
+* Explore and evaluate models from providers like Anthropic, OpenAI, and GitHub
+* Run models locally using ONNX and Ollama for privacy and cost control
+* Build and test agents with prompt generation and MCP tool integrations
+* Convert and optimize models for deployment across different hardware configurations
 
 ## Install and setup
 
-You can install the AI Toolkit from the Extensions view in VS Code:
+### Quick Installation
+
+The fastest way to get started is through the VS Code Extensions marketplace:
 
 > <a class="install-extension-btn" href="vscode:extension/ms-windows-ai-studio.windows-ai-studio">Install the AI Toolkit for VS Code</a>
 
-You can switch the installation between the formal released version for stable features and pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
+### Manual Installation
```

Comment:
This is quite an elaborate section to install an extension. Do you need so much detail in the overview page?